index.js

```diff
@@ -66,12 +73,34 @@ function concatEnvSchemas(schema, otherSchema) {
   }
 }
 
+function getCustomHeaders(headersKeyToProxy, headers) {
+  return headersKeyToProxy.reduce((acc, headerKey) => {
+    const headerValue = headers[headerKey]
+    return {
+      ...acc,
+      ...headerValue ? { [headerKey]: headerValue } : null,
```

Comment:
`null` value is acceptable? Is there a test for this case?

Reply:
We should skip only undefined and empty string value?

Reply:
```js
if (!headers.hasOwnProperty(headerKey)) return acc
```
?

Reply:
Fixed in https://github.com/mia-platform/custom-plugin-lib/pull/15/commits/f1b487d6fe35bfb27f1e49a19cbf0e2d39361d47